installer/dev/InstallActivityContext.cpp

```diff
@@ -52,4 +52,13 @@ void WindowsAppRuntimeInstaller::InstallActivity::Context::SetLastFailure(const
     {
         m_lastFailure.message.clear();
     }
+
+    if (failure.pszCode)
+    {
+        m_lastFailure.code = *failure.pszCode;
```

Comment:
No lock protecting member variable access

Reply:
Installer is an executable and a process on it's own. Hence, there is no concern of simultaneous multiple entrance to this code block that we need to protect against.

Reply:
Gotcha. Understood! @DrusTheAxe Any concerns with not having locks here? My only concern is if the APIs are somehow misused outside the installer space (Not sure if that is possible today)

Reply:
Installer doesn't expose any APIs. It's a self-contained executable.